libtransmission/torrent.cc

```diff
@@ -1067,6 +1067,7 @@ void torrentInit(tr_torrent* tor, tr_ctor const* ctor)
     tor->bandwidth_.setPriority(tr_ctorGetBandwidthPriority(ctor));
     tor->error = TR_STAT_OK;
     tor->finishedSeedingByIdle = false;
+    tor->sequentialDownload = false;
```

Comment:
This line can be removed; `sequentialDownload` is already set to `false` in the torrent constructor.